packages/auth-providers/dbAuth/setup/src/templates/web/auth.webAuthn.ts.template

```diff
@@ -1,4 +1,4 @@
 import { createDbAuth } from '@redwoodjs/auth-dbauth-web'
 import WebAuthnClient from '@redwoodjs/@redwoodjs/auth-dbauth-web/webAuthn'
 
-export const { AuthProvider, useAuth } = createDbAuth(WebAuthnClient)
+export const { AuthProvider, useAuth } = createDbAuth(new WebAuthnClient())
```

Comment:
We never did this before:

```jsx
import { AuthProvider } from '@redwoodjs/auth'
import WebAuthnClient from '@redwoodjs/auth/webAuthn'
import { FatalErrorBoundary, RedwoodProvider } from '@redwoodjs/web'
import { RedwoodApolloProvider } from '@redwoodjs/web/apollo'

import FatalErrorPage from 'src/pages/FatalErrorPage'
import Routes from 'src/Routes'

import './index.css'

const App = () => (
  <FatalErrorBoundary page={FatalErrorPage}>
    <RedwoodProvider titleTemplate="%PageTitle | %AppTitle">
      <AuthProvider type="dbAuth" client={WebAuthnClient}>
        <RedwoodApolloProvider>
          <Routes />
        </RedwoodApolloProvider>
      </AuthProvider>
    </RedwoodProvider>
  </FatalErrorBoundary>
)

export default App

```

I think because `WebAuthnClient` isn't actually a class; looks like was (and still is) a collection of functions:

- was: https://github.com/redwoodjs/redwood/blob/next/packages/auth/src/webAuthn/index.ts#L196
- still is: https://github.com/redwoodjs/redwood/blob/main/packages/auth-providers/dbAuth/web/src/webAuthn.ts#L196

So we should probably rename this to `webAuthnClient`?

Reply:
Making it a class is one of the changes introduced in this PR
![image](https://user-images.githubusercontent.com/30793/206270859-b8f98dc7-9c64-4149-99f4-0c8ce70782b2.png)

Reply:
Oh sorry about that

Reply:
And the reason is I wanted to be able to have a "set"-method to set the custom api url (and a place to store it, as a member variable)

Very much open to any name changes or whatever to make it all more clear